pandas/src/parser/tokenizer.c

```diff
@@ -1675,23 +1483,20 @@ double xstrtod(const char *str, char **endptr, char decimal,
   num_digits = 0;
   num_decimals = 0;
 
-  // Process string of digits
-  while (isdigit(*p))
-  {
+  // Process string of digits.
```

Comment:
@jreback : I cannot find any style checker.  I also cannot find any Google-related C style guideline.  But if I had to choose a C style guideline to follow, might I suggest that of the Linux kernel (see <a href="https://www.kernel.org/doc/Documentation/CodingStyle">here</a>) :smile: